cli/internal/action/checkPR.go

```diff
@@ -0,0 +1,487 @@
+package action
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"html"
+	"hyaline/internal/check"
+	"hyaline/internal/code"
+	"hyaline/internal/config"
+	"hyaline/internal/docs"
+	"hyaline/internal/github"
+	"hyaline/internal/sqlite"
+	"log/slog"
+	"os"
+	"path/filepath"
+	"reflect"
+	"slices"
+	"sort"
+	"strings"
+)
+
+type CheckPRArgs struct {
+	Config        string
+	Documentation string
+	PullRequest   string
+	Issues        []string
+	Output        string
+}
+
+type CheckPRComment struct {
+	Sha             string                         `json:"sha"`
```

Comment:
Along with the comment about matching outputs between the check commands, we could also update the check output to include the sha in both places.

And riffing off of that idea, we could extend this to return both the base and head sha used in the code diff for recommendations.